dmscripts/export_framework_results_reasons.py

```diff
@@ -194,6 +161,14 @@ def create_row(self, record):
             contact_details(record)
 
 
+def get_questions_numbers_from_framework(framework_slug, content_loader):
```

Comment:
:scream: I tried to look at this so many times up to now, sorry it's taken so long.

Trying to run this it breaks here for me, not 100% sure why without digging in a bit more:

```
Traceback (most recent call last):
  File "./scripts/export-framework-results-reasons.py", line 57, in <module>
    supplier_ids
  File "./dmscripts/export_framework_results_reasons.py", line 184, in export_suppliers
    questions_numbers = get_questions_numbers_from_framework(framework_slug, content_loader)
  File "./dmscripts/export_framework_results_reasons.py", line 168, in get_questions_numbers_from_framework
    for question in chain.from_iterable(content_loader.get_manifest(framework_slug, 'declaration'))
  File "./dmscripts/export_framework_results_reasons.py", line 167, in <dictcomp>
    question.id: question.number
  File "/Users/kevinkeenoy/gdsworkspace/digitalmarketplace-scripts/venv/lib/python2.7/site-packages/dmcontent/content_loader.py", line 176, in __getitem__
    return getattr(self, key)
TypeError: getattr(): attribute name must be string
```